test/jasmine/tests/pie_test.js

```diff
@@ -376,10 +356,37 @@ describe('Test event property of interactions on a pie plot:', function() {
     beforeEach(function() {
         gd = createGraphDiv();
         mockCopy = Lib.extendDeep({}, mock);
+        Lib.extendFlat(mockCopy.data[0], {
+            ids: ['marge', 'homer', 'bart', 'lisa', 'maggie'],
+            customdata: [{1: 2}, {3: 4}, {5: 6}, {7: 8}, {9: 10}]
+        });
     });
 
     afterEach(destroyGraphDiv);
 
+    function checkEventData(data) {
+        var point = data.points[0];
+
+        expect(point.curveNumber).toBe(0);
+        expect(point.pointNumber).toBe(4);
+        expect(point.pointNumbers).toEqual([4]);
+        expect(point.data).toBe(gd.data[0]);
+        expect(point.fullData).toBe(gd._fullData[0]);
+        expect(point.label).toBe('4');
+        expect(point.value).toBe(5);
+        expect(point.color).toBe('#1f77b4');
+        expect(point.id).toEqual(['maggie']);
+        expect(point.customdata).toEqual([{9: 10}]);
+
+        // for backward compat - i/v to be removed at some point?
```

Comment:
Blocking: yeah, we should remove that in `v2` - would you mind adding it to #420 ?

Reply:
commented https://github.com/plotly/plotly.js/issues/420#issuecomment-339475968